vm/vmcore/src/synic.rs

```diff
@@ -122,3 +135,41 @@ pub trait GuestMessagePort: Send + Sync + Inspect {
     /// Changes the virtual processor to which messages are sent.
     fn set_target_vp(&mut self, vp: u32) -> Result<(), HypervisorError>;
 }
+
+/// Represents the GPA of the outgoing and incoming monitor pages.
+#[derive(Clone, Copy, PartialEq, Eq, Debug, Default, Inspect)]
+pub struct MonitorPageGpas {
+    /// The GPA of the incoming monitor page.
+    #[inspect(hex)]
+    pub parent_to_child: u64,
+    /// The GPA of the outgoing monitor page.
+    #[inspect(hex)]
+    pub child_to_parent: u64,
+}
+
+/// Provides information about monitor usage for a synic event port.
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub struct MonitorInfo {
+    monitor_id: MonitorId,
```

Comment:
Why not just have `pub` fields and remove the accessors? It doesn't seem like there are internal invariants here that need to be maintained.